<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_funct</class>
 <widget class="QDialog" name="DB_funct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图书管理系统 V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bmp.qrc">
    <normaloff>:/bmp/C:/Users/Lucas/Desktop/DBP/books.ico</normaloff>:/bmp/C:/Users/Lucas/Desktop/DBP/books.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButton_in">
     <property name="text">
      <string>图书入库</string>
     </property>
     <property name="icon">
      <iconset resource="bmp.qrc">
       <normaloff>:/bmp/bmp/in.ico</normaloff>:/bmp/bmp/in.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>61</width>
       <height>61</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolButton_borrow">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="bmp.qrc">
       <normaloff>:/bmp/bmp/图像 5.bmp</normaloff>:/bmp/bmp/图像 5.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>61</width>
       <height>61</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButton_add">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="bmp.qrc">
       <normaloff>:/bmp/bmp/Add user.ico</normaloff>:/bmp/bmp/Add user.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>61</width>
       <height>61</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolButton_find">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="bmp.qrc">
       <normaloff>:/bmp/bmp/search.ico</normaloff>:/bmp/bmp/search.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>61</width>
       <height>61</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="toolButton_return">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="bmp.qrc">
       <normaloff>:/bmp/bmp/图像 4.bmp</normaloff>:/bmp/bmp/图像 4.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>61</width>
       <height>61</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButton_del">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="bmp.qrc">
       <normaloff>:/bmp/bmp/Remove user.ico</normaloff>:/bmp/bmp/Remove user.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>61</width>
       <height>61</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_logout">
       <property name="text">
        <string>注销</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_exit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="bmp.qrc"/>
 </resources>
 <connections/>
</ui>
